frontend/packages/dev-console/src/components/health-checks/HealthCheckProbe.tsx

```diff
@@ -111,7 +112,7 @@ const HealthCheckProbe: React.FC<HealthCheckProbeProps> = ({ probeType }) => {
 
   return (
     <>
-      <div className="co-section-heading-tertiary odc-heath-check-probe__formTitle">
+      <TertiaryHeading className="odc-heath-check-probe__formTitle">
```

Comment:
The `<h3>`s on the Add Health Checks form now have excess top and bottom spacing. Because they are within `pf-v6-c-form` grid which sets a grid-gap. Maybe have a `noMargin` prop to `TertiaryHeading`
<img width="699" alt="Screenshot 2025-02-07 at 11 21 46 AM" src="https://github.com/user-attachments/assets/bd42d74d-46fd-4124-ae09-96499b369488" />

Reply:
After
<img width="1196" alt="Screenshot 2025-02-07 at 1 15 13 PM" src="https://github.com/user-attachments/assets/4c4226f2-4d6a-4391-bab1-7292c9d3615e" />